skimage/measure/_ccomp.pyx

```diff
@@ -389,14 +386,15 @@ def label(input, neighbors=None, background=None, return_num=False,
         **Deprecated, use ``connectivity`` instead.**
     background : int, optional
         Consider all pixels with this value as background pixels, and label
-        them as -1. (Note: background pixels will be labeled as 0 starting with
-        version 0.12).
+        them as 0. If ``None``, 0-valued pixels are considered as background
```

Comment:
:+1: for defaulting to 0. I usually like the `kwarg=None` pattern, but in this case an explicit default is more immediately informative to a naive user.